common/modules/aws-gcp-dns/aws-dns-zone/main.tf

```diff
@@ -10,15 +12,15 @@ resource "aws_route53_zone" "main" {
   name = "${var.record_name}.gpii.net"
   lifecycle   {
      prevent_destroy = "true"
-  } 
+  }
   tags {
     Terraform = true
   }
 }
 
 
 resource "aws_route53_record" "main_ns" {
-  zone_id = "Z26C1YEN96KOGI"  # Unmanaged route53 zone for gpii.net
+  zone_id = "${var.aws_zone_id}"  # Unmanaged route53 zone for gpii.net
```

Comment:
The comment is about the value, not the code, so can you move the comment to where you moved the value?